File: app/src/main/java/nl/koenhabets/yahtzeescore/MyBackupAgent.java
package nl.koenhabets.yahtzeescore;

import android.app.backup.BackupAgentHelper;
import android.app.backup.SharedPreferencesBackupHelper;

public class MyBackupAgent extends BackupAgentHelper {

    // The name of the SharedPreferences file
    static final String PREFS = "nl.koenhabets.yahtzeescore";


    @Override
    public void onCreate() {
        SharedPreferencesBackupHelper helper = new SharedPreferencesBackupHelper(this, PREFS);
        addHelper("scoresSaved", helper);
    }

}

File: app/src/main/java/nl/koenhabets/yahtzeescore/MyBackupAgent.kt
package nl.koenhabets.yahtzeescore

import android.app.backup.BackupAgentHelper
import android.app.backup.SharedPreferencesBackupHelper

class MyBackupAgent : BackupAgentHelper() {
    override fun onCreate() {
        val helper = SharedPreferencesBackupHelper(this, PREFS)
        addHelper("scoresSaved", helper)
    }

    companion object {
        // The name of the SharedPreferences file
        const val PREFS = "nl.koenhabets.yahtzeescore"
    }
}